README.md

```diff
@@ -36,15 +36,27 @@ To run a local zip file through the checks:
 
 `gscan /path/to/theme.zip -z`
 
-You can also pass a version. Currently supported is `--v1` or `-1`, which will check the theme
-for 1.0.0 requirements. By default, GScan will always check for the latest version:
+You can also pass a version. Currently supported are:
+- `--v1` or `-1`
+- `--v2` or `-2`
+- `--v3` or `-3`
+
+Above will check the theme for according 1.0, 2.0, 3.0 requirements. By default, GScan will always check for the latest (2.0) version:
 
 `gscan /path/to/theme.zip -z1`
 
 or
 
 `gscan /path/to/theme/directory --v1`
 
+or
+
+`gscan /path/to/theme/directory --v3`
+
+Another accepted parameter is `--canary` which checks for upcoming (3.0) version:
+
+`gscan /path/to/theme/directory --canary`
+
```

Comment:
My suggestion for the section:
----

### 2. CLI usage
Install using yarn / npm:
`yarn global add gscan` /  `npm install -g gscan`

To run a local directory through the checks:
`gscan /path/to/theme/directory`

To run a local zip file through the checks:
`gscan /path/to/theme.zip -z`

By default, GScan scans themes for the latest Ghost version compatibility. You can also specify a Ghost version by using the following parameters (for Ghost 1.0, 2.0 and 3.0):

`-1` or `--v1`
`-2` or `--v2`
`-3` or `--v3`

Use the `--canary` parameter to check for the upcoming Ghost version.

Examples:

`gscan /path/to/theme.zip -z1` -- scan a theme in a zip file for Ghost 1.0 compatibility
`gscan /path/to/theme/directory --v2` -- scan a theme in a directory for Ghost 2.0 compatibility
`gscan /path/to/theme/directory --canary` -- scan a theme for the upcoming version of Ghost

Reply:
@JohnONolan could you please doublecheck if the above makes sense? thank you

Reply:
@peterzimon @gargol does this read better do you think?

### 2. CLI usage

Install using `yarn` or `npm`:
```
yarn global add gscan
```

To check a theme locally:
```
gscan /path/to/theme/directory
```

To check a theme locally within a zip file:
```
gscan /path/to/theme.zip -z
```

By default GScan checks themes against the latest version of Ghost. Flags can be used to check themes against a specific version of Ghost:
- `-1` or `--v1`
- `-2` or `--v2`
- `-3` or `--v3`

Use the `--canary` flag to check a theme against an upcoming version of Ghost.
```
gscan /path/to/theme/directory --canary
```

Reply:
@daviddarnes the code sections - "```", read a bit better. Not a fan of those bullet lists tbh :) soft line break seems less distracting when there's not much content in the list to show.

Reply:
@gargol I'm cool with that 😄

Reply:
the other bit that doesn't hold up with code sections - "```", is the npm/yarn installation section... think for consistency sake and to keep the "examples" section will stick to @peterzimon 's style. welcome to PR again if it feels off.

Reply:
Fair enough, I'm not precious about the formatting. I only put yarn as it's a common pattern we do in the docs, we mention npm briefly but we use yarn for the majority of the examples